packages/asc-ui/src/components/ContextMenu/ContextMenuButton.tsx

```diff
@@ -12,22 +11,11 @@ const ContextMenuButton = ({
   ...otherProps
 }: any) => {
   return (
-    <AscCore.ContextMenu.MenuButton {...{ id }} {...otherProps}>
+    <ContextMenuStyle.ContextMenuButtonStyle {...{ id }} {...otherProps}>
```

Comment:
can this not be destructured? Otherwise use something like 
const ContextMenuButtonStyle = ContextMenuStyle.ContextMenuButtonStyle
for readability